packages/layer-selector/src/LayerSelector.jsx

```diff
@@ -316,14 +340,12 @@ const LayerSelector = (props) => {
       ) || [];
     let overlays = props.overlays || [];
 
-    let hasDefaultSelection = false;
```

Comment:
**question**: do default base map and overlay selections still work?

Reply:
Yes. I added some stories for testing.